.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,64 @@
+name: Deploy Storybook
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - v*
+      - '!v*-beta.*'
```

Comment:
Note that I'm now excluding beta tags, so that the Storybook gets deployed only for official releases. We can still trigger the workflow manually if need be.